ebean-core/src/main/java/io/ebeaninternal/server/core/DScriptRunner.java

```diff
@@ -66,8 +66,9 @@ private String content(URL resource) {
       throw new IllegalArgumentException("resource is null?");
     }
 
-    try (InputStream inputStream = UrlHelper.openNoCache(resource)) {
-      return readContent(new InputStreamReader(inputStream));
+    try (InputStream inputStream = UrlHelper.openNoCache(resource);
+        Reader reader = IOUtils.newReader(inputStream)) {
```

Comment:
Note to review indentation

Reply:
@rbygrave Thanks for the comments! I have updated the PR